code/WorkInProgress/Venseer/RemoteCyborg.dm

```diff
@@ -0,0 +1,304 @@
+  desc = "A off-putting looking board. Instead of standard green and yellow, it is black with red circuits, there is a small antenna on the side."
+  icon = 'icons/obj/RemoteCyborg.dmi'
+  icon_state = "syndie_board_01"
+  w_class = W_CLASS_SMALL
+  flags = FPRINT | NOBLUDGEON
+  var/frequency = "syndicate"
+  var/active = 0
+  var/mob/living/silicon/cyborg
+  var/inUse
+  var/controller = null
+
+/obj/item/device/syndicate_remote_cyborg_control_board/afterattack(atom/target, mob/user as mob, proximity_flag)
+  if(!istype(target, /obj/item/robot_parts/robot_suit))
+    return
+  var/obj/item/robot_parts/robot_suit/robot_suit = target
+  var/turf/T = get_turf(robot_suit)
+  if(robot_suit.check_completion())
+    if(!istype(T,/turf))
+      to_chat(user, "<span class='warning'>You can't put the motherboard in, the frame has to be standing on the ground to be perfectly precise.</span>")
+      return
+
+    var/mob/living/silicon/robot/remote_control_robot/robot = new /mob/living/silicon/robot/remote_control_robot(get_turf(target), unfinished = 1)
+    robot.invisibility = 0
+    robot.custom_name = robot_suit.created_name
+    robot.updatename("Default")
+    robot.job = "Cyborg"
+    robot.cell = robot_suit.chest.cell
+    robot.cell.loc = robot
+    if(robot.cell)
+      var/datum/robot_component/cell_component = robot.components["power cell"]
+      cell_component.wrapped = robot.cell
+      cell_component.installed = 1
+    robot.mmi = src
+    robot.verbs += /mob/living/silicon/robot/remote_control_robot/proc/Exit_robot
+    src.cyborg = robot
+    src.active = 1
+    rc_control_boards += src
+    qdel(robot_suit)
+    robot.emagged = 1
+    user.drop_item(src, robot)
+
+  else
+    to_chat(user, "<span class='notice'>This robot does not seem to be done. You need all parts to be in place in order to insert the remote control board.</span>")
+
+/*
+    END CONTROL BOARD SECTION
+*/
+
+/*
+    START CONTROLLER SECTION
+*/
+/obj/item/device/syndicate_controller
+  name = "R.C. Controller"
+  desc = "A Remote Cyborg Controller."
+  icon_state = "handtv"
+  flags = FPRINT
+  w_class = W_CLASS_TINY
+  var/obj/item/device/syndicate_remote_cyborg_camera/current_camera
+  var/obj/item/device/syndicate_remote_cyborg_control_board/current_board
+  var/frequency = "syndicate"
+  var/user_ckey = ""
+  var/mob/living/carbon/user_body
+  var/active = 0
+
+/obj/item/device/syndicate_controller/New()
+  processing_objects.Add(src)
+
+/obj/item/device/syndicate_controller/Destroy()
+  processing_objects.Remove(src)
+
+/obj/item/device/syndicate_controller/process()
+  if(src.active == 0 || !(src.current_board))
+    return
+  else
+    var/mob/living/carbon/user = src.user_body
+    if ( src.loc != user || user.canmove == 0 || (user.active_hand != 1 && user.active_hand != 2) || user.held_items.Find(src) == 0 || user.blinded || !src.current_board || !src.current_board.active || src.current_board.cyborg.isDead() || user.monkeyizing && user)
```

Comment:
That active_hand index comparison check is terrible, you can use get_active_hand(), which you don't need to do because you're checking held items immediately after anyways.